model/implementation/DeliveryExecutionStateService.php

```diff
@@ -50,6 +49,16 @@ class DeliveryExecutionStateService extends ConfigurableService implements \oat\
     private $testSessionService;
 
     /**
+     * @inheritdoc
+     * @param DeliveryExecutionInterface $deliveryExecution
+     * @param $state
+     */
+    public function setState(DeliveryExecutionInterface $deliveryExecution, $state)
+    {
+        $deliveryExecution->setState($state);
```

Comment:
This is temporary default implementation.
Logic of changing state i will implement in further PRs.